### PR TITLE
refs platform/#2978: implement init filesystem and seed archives feature

### PR DESCRIPTION
**Pull request by Monska85**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,86 @@
...
+
+## Bucket initialization
+
+You can use two different methods to initialize the bucket: the `FileSystem` initialization and the `Seed` initialization.
+
+## FileSystem initialization
+
+If the `INITFILESYSTEM_FOLDER` path is not empty, the files in the folder are copied to the `BUCKET_ROOT` destination folder as a startup filesystem for the MinIO server. **The folder must contain a valid MinIO filesystem structure.** **ATTENTION**: when the `INITFILESYSTEM_FOLDER` is used, the startup process takes care of checking that the `BUCKET_NAME` is configured accordingly with the final MinIO server configuration, performing the filesystem check (the `BUCKET_NAME` folder is present in the `BUCKET_ROOT` folder) and the MinIO client check (the `mc ls ${MC_ALIAS}/${BUCKET_NAME}` command does not return an error).
+
+If this initialization method is used, **no other initialization actions are performed**. The final bucket content is only the data present in the `INITFILESYSTEM_FOLDER` folder.
```
Comment (paolomainardi) [inline README.md:15]: I am wondering if we should support both methods, seed files +  bucket files, they shouldn't collide and can be processed in a chain.

File: README.md [hunk 2]
```diff
@@ -1,32 +1,86 @@
...
+
+**The supported archive formats are `.zip`, `.tar`, `.tar.gz`, `.tar.bz2` and `.tar.xz`.**
+
+### Process the seed files
+
+If the `INITFILES_FOLDER` path is not empty, the files in the folder are uploaded in the bucket defined by the `BUCKET_NAME` environment variable using the MinIO client (`mc cp --recursive`). As for the archives, if the `INITFILES_FOLDER` contains files with the same name of one or more files already present in the bucket (processed by the `INITARCHIVES_FOLDER`), the files in the `INITFILES_FOLDER` will overwrite the files in the bucket. If the bucket has a versioning enabled, the previous versions of the files are preserved.
```
Comment (paolomainardi) [inline README.md:33]: I would chance FOLDER with DIR

Comment (Monska85) [reply README.md]: Done.

File: README.md (hunk 1 above)
Comment (Monska85) [reply README.md]: Implemented. [This](https://github.com/sparkfabrik/Docker-MinIO/pull/14/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R7) is the new _seed_ logic.

If the bucket exists at the startup, no seed process will be executed. In other words, if  the `/data` folder is a valid MinIO filesystem with an already created bucket, nothing happens. To test this scenario, you can mount a local data folder in `/data` container mount point and exec `make start` twice. At the first run the container will seed the mounted `/data` folder, at the second run the container will skip the initialization process.

**Review by paolomainardi — APPROVED** [review]

**Pull request merged**